apigentools/commands/init.py

```diff
@@ -102,7 +102,7 @@ def run(self):
             )
             if not os.path.exists(config_file):
                 with open(config_file, "w") as f:
-                    json.dump(self.CONFIG_FILE_JSON, f, indent=4)
+                    yaml.dump(self.CONFIG_FILE_JSON, f, indent=4)
```

Comment:
I would say 2 for yaml files.
```suggestion
                    yaml.dump(self.CONFIG_FILE_JSON, f, indent=2)
```